tg/models.py

```diff
@@ -287,7 +288,8 @@ def __init__(self, tg: Tdlib):
         self.msgs: Dict[int, List[Dict]] = defaultdict(list)
         self.current_msgs: Dict[int, int] = defaultdict(int)
         self.not_found: Set[int] = set()
-        self.msg_ids: Dict[int, Set[int]] = defaultdict(set)
+        self.msg_ids: Dict[int, Dict[int, int]] = defaultdict(dict)
```

Comment:
naming could be improved. it stores not only msg IDs. could be renamed to `msg_storage` like it is done in https://github.com/zhukov/webogram/blob/master/app/js/messages_manager.js#L13

Reply:
oh, ignore me, i can see this dict stores IDs only, but not messages itself

Reply:
ok, but still naming is misleading, could bee renamed to `msg_idxes`